### PR TITLE
Use templated index for smoke tests.

### PR DESCRIPTION
**Pull request by jmcarp**
Goes with https://github.com/cloudfoundry-community/logsearch-smoke-tests/pull/2. Note--will need to update submodule before merge.

### CODE REVIEW

**Comment by jmcarp** [comment]
Updated tests submodule and changelog.

File: jobs/smoke-tests/spec [hunk 1]
```diff
@@ -29,4 +29,4 @@ properties:
     default: 9200
   smoke_tests.elasticsearch_master.app_index:
     description: App index name
-    default: logs-app
+    default: 'logs-app-{{.Org}}-{{.Space}}-{{.Time.Format "2006-01-02"}}'
```
Comment (Infra-Red) [inline jobs/smoke-tests/spec]: @jmcarp Default app index also need to be updated to `logs-app-{{.Org}}-{{.Space}}-{{.Time.Format \"2006.01.02\"}}`. Everything else lgtm.

**Review by Infra-Red — CHANGES_REQUESTED** [review]

**Comment by jmcarp** [comment]
Updated!

**Comment by Infra-Red** [comment]
@jmcarp I've applied the patch and got `ERROR: invalid character '2' after object key:value pair` error. I guess that escaping also need to be included in the index pattern.

**Comment by jmcarp** [comment]
Thanks, updated to escape double quotes.

**Review by Infra-Red — APPROVED** [review]

**Pull request merged**